src/schema/fair.ts

```diff
@@ -47,6 +49,44 @@ const FairType = new GraphQLObjectType({
   name: "Fair",
   fields: () => ({
     ...GravityIDFields,
+    artistsConnection: {
```

Comment:
I believe [our preference](https://github.com/artsy/README/blob/f8cc033ec7c71f7f19605a04f0c8b8d5d21b9d7f/playbooks/graphql-schema-design.md#how-to-model-our-graph) now is to just name this field `artists`, so long as a field by that name does not already exist.